src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -278,6 +283,10 @@ def get_obs_data(bufr, profile_meta_data, add_qc, record_number=None):
         for k in obs_data.keys():
             obs_data[k] = obs_data[k][good]
 
+    if addLSW:  
+        lsw = bang_err/bang * 100
+        obs_data[('localSpectralWidth', "ObsError")] = assign_values(lsw)
```

Comment:
Would this cause an issue if _bang_ is zero?

Reply:
Thanks for pointing that out, I fixed it